aten/src/ATen/native/UnaryOps.cpp

```diff
@@ -188,7 +188,15 @@ Tensor imag(const Tensor& self) {
 }
 
 Tensor& conj_out(Tensor& result, const Tensor& self) { return unary_op_impl_out(result, self, conj_stub); }
```

Comment:
What about this one?
What happens if you try to use autograd with it? Does it still raise a nice error?

Reply:
I don't follow. it's only called through `at::_conj` which has a definition in `derivatives.yaml`. how else can you use it with autograd?

Reply:
This one is not called from `_conj` no.
And the user can call it directly by doing `a.conj(out=b)`.
And this should raise an error if either a or b requires grad.